src/openrct2-ui/drawing/engines/opengl/TextureCache.h

```diff
@@ -202,7 +203,7 @@ class TextureCache final
     std::vector<Atlas> _atlases;
     std::unordered_map<GlyphId, AtlasTextureInfo, GlyphId::Hash, GlyphId::Equal> _glyphTextureMap;
     std::vector<AtlasTextureInfo> _textureCache;
-    std::array<uint32_t, 0x7FFFF> _indexMap;
+    std::array<uint32_t, SPR_IMAGE_LIST_END> _indexMap;
```

Comment:
Previously, `SPR_IMAGE_LIST_END` was `0x7FFFE` and this was `0x7FFFF`, so shouldn’t now be `SPR_IMAGE_LIST_END + 1`?

Reply:
I don't think so. `SPR_IMAGE_LIST_END` is now not part of the inclusive range of available image IDs. So it is already now implicity `lastid + 1`.